### PR TITLE
ts/bubblelegend-composition

### PR DESCRIPTION
**Pull request by marvin19**
Improved BubbleLegendComposition.

### CODE REVIEW

**Comment by bre1470** [comment]
`$(...).datepicker is not a function` error in visual sample `Point.mouseOver (samples/**********/blog/stock-calendar/demo.js.preprocessed.js`. Det ser ut lik noko jQuery feil. 🤔

File: ts/Series/MapBubble/MapBubbleSeries.ts [hunk 1]
```diff
@@ -51,6 +53,7 @@ class MapBubbleSeries extends BubbleSeries {
      *  Static Properties
      *
      * */
+    public static compose = BubbleLegendComposition.compose;
```
Comment (bre1470) [inline ts/Series/MapBubble/MapBubbleSeries.ts:56]: This should use `BubbleSeries.compose`.

**Review by bre1470 — CHANGES_REQUESTED** [review]

**Comment by TorsteinHonsi** [comment]
> This looks like some jQuery failure

Yes, it's from a different PR. I've fixed it now I believe.

File: ts/Series/MapBubble/MapBubbleSeries.ts (hunk 1 above)
Comment (marvin19) [reply ts/Series/MapBubble/MapBubbleSeries.ts:56]: On it!

**Review by bre1470 — APPROVED** [review]

**Pull request merged**